rules/1.yml

```diff
@@ -14,3 +14,5 @@ groups:
     labels:
       foo: bar
       bar: foo
+
```

Comment:
:stop_sign: **Fatal** reported by [pint](https://cloudflare.github.io/pint/) **yaml/parse** check.

------

YAML parser returned an error when reading this file: `error at line 3: invalid group key extra`.

pint cannot read this file because YAML parser returned an error.
This usually means that you have an indention error or the file doesn't have the YAML structure required by Prometheus for [recording](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) and [alerting](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) rules.
If this file is a template that will be rendered into valid YAML then you can instruct pint to ignore some lines using comments, see [pint docs](https://cloudflare.github.io/pint/ignoring.html).


------

:information_source: To see documentation covering this check and instructions on how to resolve it [click here](https://cloudflare.github.io/pint/checks/yaml/parse.html).